src/test/kotlin/com/igorwojda/list/countuniquevalues/challenge.kt

```diff
@@ -4,7 +4,16 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countUniqueValues(list: List<Int>): Int {
-    TODO("not implemented")
+    if (list.isEmpty())
+        return 0
+    // In the requirement it is specified that the list is sorted, but it's not always the case
+    val sortedList = list.sorted()
```

Comment:
This implementation is interesting, but how much time/complexity are you wasting to sort it if the list is large?